### PR TITLE
Fixes #483, Added settings tab with a drop down

### LINKED ISSUE #483: Wrong position of Settings Tab and absence of drop-down menu
**Actual Behaviour**
The settings tab is in the footer of Susper.

**Expected Behaviour**
Google has the settings bar on the top, next to Tools.

**Screenshots of the issue**
image

image

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixes issue #483 

Changes: Added settings tab with a drop down, also changed CSS of the tools option, to accommodate settings, made it responsive

Demo Link:[Here](https://marauderer97.github.io/susper.com)

Screenshots for the change: 

image
image

### CODE REVIEW

**Comment by harshit98** [comment]
@Marauderer97 

screenshot from 2017-06-16 16-53-46

screenshot from 2017-06-16 16-54-08

screenshot from 2017-06-16 16-54-21

screenshot from 2017-06-16 16-54-34

**Comment by Marauderer97** [comment]
@harshit98 Please check now
image

File: src/app/footer-navbar/footer-navbar.component.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
             <a routerLink="/advancedsearch" routerLinkActive="active">Advanced Search</a>
             <a routerLink="/terms" routerLinkActive="active">Terms</a>
             <a routerLink="/contact" routerLinkActive="active">Contact</a>
-          <a routerLink="/preferences" routerLinkActive="active">Settings</a>
+           <a routerLink="/preferences" routerLinkActive="active">Settings</a>
```
Comment (nikhilrayaprolu) [inline src/app/footer-navbar/footer-navbar.component.html]: there is a drop down at the bottom too on google

**Comment by Marauderer97** [comment]
@nikhilrayaprolu
Please check now
image

**Comment by Marauderer97** [comment]
@mariobehling Please check if satisfactory now,

- I have aligned the text to the centre

- I have pushed the drop-down below, increasing space between settings and drop-down

- Changed padding to make is same on top and bottom

**Comment by mariobehling** [comment]
@Marauderer97 Please read change requests carefully.

There was no request to align to the centre, but a request to align in the middle (which centering between top and bottom)

**Comment by mariobehling** [comment]
Actually just look at Google. 

One of the reasons we are following Google is, because we apparently do not have a lot of expertise in design. So, instead of re-inventing a design, we just follow. However, I see that it is still difficult to pay attention to detail. Let's not take too much time of people reviewing simple items and please look into details before making a PR in future.

**Comment by Marauderer97** [comment]
Changes made: 
1. Aligned text back
2. Changed padding, border-radius, background colour and font colour to match Google exactly, as much as I could, trying to pay attention to maximum detail(did not do so previously because of Tools menu, now Tools and Settings drop-down look different, perhaps we can raise an issue on Tools drop-down later)
image

**Comment by mariobehling** [comment]
> perhaps we can raise an issue on Tools drop-down later

Please do. No need to wait for me to confirm. Just continue doing what is required to make this project work and look smooth.

Please resolve merge conflicts.

**Pull request merged**